pkg/servicedisc/client.go

```diff
@@ -230,8 +230,8 @@ func (c *HTTPClient) postEntry(ctx context.Context) (Service, error) {
 	return entry, err
 }
 
-// DeleteEntry calls 'DELETE /api/services/{entry_addr}'.
-func (c *HTTPClient) DeleteEntry(ctx context.Context) (err error) {
+// DeregisterEntry calls 'DELETE /api/services/{entry_addr}'.
```

Comment:
Maybe `DeleteEntry` is clearer. Sorry for not using the correct term before.

Reply:
Done.